docs/arch/adr-001.md

```diff
@@ -13,7 +13,8 @@ The [Discord API documentation](https://discordapp.com/developers/docs/intro) is
 the JSON structures it returns and accepts.
 
 Code generation is often difficult to use with IDEs.
-Code is often referring to classes that will not be committed, so have to be generated by the IDE.
+Code is often referring to classes that will not be committed, so we depend upon the IDE to generate these classes
+so that useful messages can be presented to the developer.
```

Comment:
messages? I feel like we're more generating these classes to provide utility, not sure I understand the definition for `messages` you're using here.

Reply:
In this context, it's messages by the IDE to the developer (e.g., this method doesn't exist)

Reply:
Hmm okay, it's a bit odd in my opinion reading it back a few times but in context it makes sense.